src/state_management.h

```diff
@@ -9,13 +9,14 @@
 #include "trigger_in_out.h"
 #include "trigger_out.h"
 #include "network_handle.h"
+#include "state_client.h"
 
 namespace ara { namespace sm {
     /** @brief Class StateManagement to be used by Update and Configuration Management.
      *         Fills [SWS_SM_XXXXX]. */
     class StateManagement {
     public:
-        StateManagement();
+        StateManagement(exec::StateClient * sc);
```

Comment:
shall be 'exec::StateClient* sc' or 'exec::StateClient *sc'. Nowadays, I think the first option is common

Reply:
Done